.github/actions/setup-go/action.yaml

```
@@ -0,0 +1,24 @@
+name: Setup Go Only
+description: Lightweight Go setup without unnecessary tools
+
+inputs:
+  github_token:
+    description: GitHub token for authentication
+    required: false
+
+runs:
+  using: "composite"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)